web-animations-2/Overview.bs

```diff
@@ -2043,12 +2114,48 @@ SequenceEffect implements GroupEffectMutable;
 <h3 id="the-keyframeeffect-interfaces">The <code>KeyframeEffectReadOnly</code>
   and <code>KeyframeEffect</code> interfaces</h3>
 
-Add:
+The <a>KeyframeEffect</a> interface is modified to add the following:
 
 <pre class='idl'>
+interface KeyframeEffect : AnimationEffect {
```

Comment:
I believe this should be `partial interface`?

Reply:
Ah, TIL about `partial interface`s, thanks :)

Reply:
Done.